### DISCUSSION TITLE
Жизненный цикл компонента ⚡️ Angular с примерами кода

### DISCUSSION THREAD

**Discussion by giscus** [xsltdev/angdev.ru#39] [Announcements]
# Жизненный цикл компонента ⚡️ Angular с примерами кода

Каждый компонент имеет свой жизненный цикл (Component Lifecycle), в процессе которого вызываются ряд описывающих текущий этап методов (Angular Hooks)

https://angdev.ru/doc/component-lifecycle/

**Comment by IvanBanifatsyeu** [comment]
Очень полезно и наглядно !
